src/transformers/trainer.py

```diff
@@ -1280,12 +1280,12 @@ def ipex_optimize_model(self, model, training=False, dtype=torch.float32):
 
         if not training:
             model.eval()
-            model = ipex.optimize(model, dtype=dtype, level="O1")
+            model = ipex.optimize(model, dtype=dtype, level="O1", conv_bn_folding=False)
         else:
             if not model.training:
                 model.train()
             model, self.optimizer = ipex.optimize(
-                model, dtype=dtype, optimizer=self.optimizer, inplace=True, level="O1"
+                model, dtype=dtype, optimizer=self.optimizer, inplace=True, level="O1", conv_bn_folding=False
```

Comment:
For training, we do not have to add this flag since it is only for inference ([API doc](https://intel.github.io/intel-extension-for-pytorch/latest/tutorials/api_doc.html#ipex.optimize))